### PR TITLE
add custom page ref url

### PR DESCRIPTION
**Pull request by jljsj33**

### CODE REVIEW

**Comment by ycjcl868** [comment]
image

404

**Comment by jljsj33** [comment]
强刷，

File: docs/guide/faq.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ dumi 基于 Umi，即除了自己提供的配置项以外，还支持[所有 Umi
 
 <code src="path/to/homepage.tsx" inline />
 ```
+详细使用可参考 ant design landing 的 [use in dumi](https://landing.ant.design/docs/use/dumi-cn)
```
Comment (PeachScript) [inline docs/guide/faq.md:30]: Ant Design landing？

**Review by PeachScript — CHANGES_REQUESTED** [review]

File: docs/guide/faq.md (hunk 1 above)
Comment (jljsj33) [reply docs/guide/faq.md]: done

**Review by PeachScript — APPROVED** [review]
+1

**Pull request merged**